test/backend/openfisca/test.js

```diff
@@ -40,4 +80,24 @@ describe('openfisca generateYAMLTest', function() {
     it('contains provided output_variables', function() {
         expect(result).toInclude('valueOne: 1');
     });
+
+    if (process.env.INTEGRATION_TEST) {
```

Comment:
Can't we use something else to make sure we are in a virtual env? 
Maybe `VIRTUAL_ENV` variable?